compiler/optimizer/Dominators.cpp

```diff
@@ -110,15 +124,22 @@ TR_Dominators::TR_Dominators(TR::Compilation *c, bool post) :
             traceMsg(comp(), "Some blocks are not reachable from exit. Post-dominator info is invalid.\n");
          return;
          }
+      if (!acceptUnreachableBlocks)
+         {
+         _hasUnreachableBlocks = true;
```

Comment:
Please stick with `_isValid`, which already has the same meaning, and remove  `_hasUnreachableBlocks`